packages/web/src/components/flair/FlairHoverCardBody.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from 'react'
+
+import { Flex, Text } from '@audius/harmony'
+
+export type FlairHoverCardBodyProps = {
+  icon: ReactNode
+  amount: string
+  currency?: string
+}
+
+export const FlairHoverCardBody = ({
```

Comment:
probably best to rewrite these components as a single "AudioHoverCard" component that implements the header and body? then update UserBadges to wrap the Audio badge with "AudioHoverCard"?